<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>243</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>500</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>vision_port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="6">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="multicast_ip_input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="vision_port_input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="referee_ip_input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="referee_port_input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="yellow_port_input"/>
        </item>
        <item>
         <widget class="QLineEdit" name="blue_port_input"/>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>blue_port</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>referee_ip</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>yellow_port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>multicast_ip</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>referee_port</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 3px;
	background: rgb(234, 239, 44);
	color: black;
}
QPushButton:hover {
    background: rgba(234, 239, 44, 220);
}

QPushButton:pressed {
    background: rgba(234, 239, 44, 40);
}</string>
   </property>
   <property name="text">
    <string>Resetar Default</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius: 3px;
	background: rgb(0, 85, 255);
	color: white;
}
QPushButton:hover {
    background: rgba(0, 85, 255, 220);
}

QPushButton:pressed {
    background: rgba(0, 85, 255, 200);
}</string>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>